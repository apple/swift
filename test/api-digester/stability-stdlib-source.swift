// REQUIRES: OS=macosx
// RUN: %empty-directory(%t.tmp)
// mkdir %t.tmp/module-cache && mkdir %t.tmp/dummy.sdk
// RUN: %api-digester -diagnose-sdk -module Swift -o %t.tmp/changes.txt -module-cache-path %t.tmp/module-cache -sdk %t.tmp/dummy.sdk -avoid-location
// RUN: sed 's|/[*].*[*]/||g' %S/Outputs/stability-stdlib-source.swift.expected | sed '/^\s*$/d' | sort > %t.tmp/stability-stdlib-source.swift.expected
// RUN: sed 's|/[*].*[*]/||g' %t.tmp/changes.txt | sed '/^\s*$/d' | sort > %t.tmp/changes.txt.tmp
// RUN: diff -u %t.tmp/stability-stdlib-source.swift.expected %t.tmp/changes.txt.tmp
