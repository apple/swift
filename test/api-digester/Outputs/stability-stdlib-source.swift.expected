
/* Generic Signature Changes */

/* RawRepresentable Changes */

/* Removed Decls */
Func Collection.prefix(through:) has been removed
Func Collection.prefix(upTo:) has been removed
Func Collection.suffix(from:) has been removed
Func Sequence.filter(_:) has been removed
Func Sequence.forEach(_:) has been removed
Func Sequence.map(_:) has been removed
Func _SequenceWrapper.filter(_:) has been removed
Func _SequenceWrapper.forEach(_:) has been removed
Func _SequenceWrapper.map(_:) has been removed
Var AnyBidirectionalCollection.first has been removed
Var AnyBidirectionalCollection.last has been removed
Var AnyCollection.first has been removed
Var AnyRandomAccessCollection.first has been removed
Var AnyRandomAccessCollection.last has been removed
Var BidirectionalCollection.last has been removed
Var Collection.first has been removed
Var Dictionary.first has been removed
Var LazyCollection.first has been removed
Var LazyCollection.last has been removed
Var LazyMapCollection.first has been removed
Var LazyMapCollection.last has been removed
Var Set.first has been removed

/* Moved Decls */

/* Renamed Decls */

/* Type Changes */

/* Decl Attribute changes */

/* Protocol Requirement Changes */
